### PR TITLE
Define a suitable file structure

### PR DESCRIPTION
**Pull request by sylsirlym**
#### What does this PR do?
Define file structure
#### Description of Task to be completed?
Define the file structure
Tests BD connection in routes.py
Add a package database.py and add a DB class
#### Stories
[#158802885](https://www.pivotaltracker.com/story/show/158802885)

### CODE REVIEW

File: api-v2/models/database.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+from psycopg2.extras import RealDictCursor #Access DB fields access columns only from keys
+
+class Database(object):
+    def __init__(self, app=None):
+        self.app = app
+        if app is not None:
+            self.init_app(app)
+    
+    def init_app(self, app):
+        self.conn = connect("host='localhost' dbname='ridemyway' user='postgres' password='Secrets'")
```
Comment (esirK) [inline api-v2/models/database.py]: add these variables to a `dot-env` file. It is not a good practice to push your connection details to Github.

Comment (sylsirlym) [reply api-v2/models/database.py]: @esirK 
On it. Thanks

File: api-v2/models/models.py [hunk 2]
```diff
@@ -0,0 +1,115 @@
+users = []
```
Comment (esirK) [inline api-v2/models/models.py:1]: do you need this data structures??

File: api-v2/models/models.py [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+
+class User:
+    def __init__(self, fname, lname,email,password):
+        self.fname = fname
+        self.lname = lname
+        self.email = email
+        self.password = password
+
+    def add(self):
+        users.append(self)
```
Comment (esirK) [inline api-v2/models/models.py:14]: Should you be storing your **users** inside a `list` for **challenge 3**??

File: api-v2/models/models.py [hunk 4]
```diff
@@ -0,0 +1,115 @@
...
+
+        for user in users:
+            print(self.serialize(user))
```
Comment (esirK) [inline api-v2/models/models.py:17]: what is the purpose of this?? The `print(self.serialize(user))`

**Pull request closed without merging**